lms/djangoapps/commerce/__init__.py

```diff
@@ -13,7 +13,19 @@ def create_tracking_context(user):
     }
 
 
+def is_commerce_service_configured():
+    """
+    Return a boolean indicating whether or not configuration is present to use
```

Comment:
thanks.  since you mention this, in my next update, the return from that function will actually be a boolean too.